### PR TITLE
Update latest base images versions

### PR DESCRIPTION
**Pull request by nghiant2710**
Change-type: patch
Signed-off-by: Trong Nghia Nguyen <nghiant2710@gmail.com>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/reference/base-images/base-images.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
   - amd64
   - i386
 - Multiple Distributions:
-  - [Debian](https://www.debian.org/): jessie (8), stretch (9), buster (10), bullseye (11), and sid
-  - [Alpine](https://alpinelinux.org/): 3.10, 3.11, 3.12, 3.13 and edge
-  - [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), cosmic (18.10), disco (19.04), eoan (19.10) and focal (20.04)
-  - [Fedora](https://getfedora.org/): 30, 31, 32, 33 and 34
+  - [Debian](https://www.debian.org/): stretch (9), buster (10), bullseye (11), bookworm and sid
+  - [Alpine](https://alpinelinux.org/): 3.11, 3.12, 3.13, 3.14 and edge
+  - [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), focal (20.04), hirsute and impish
```
Comment (lurch) [inline pages/reference/base-images/base-images.md:22]: Should this mention which release-numbers `hirsute` and `impish` refer to?

Comment (nghiant2710) [reply pages/reference/base-images/base-images.md]: @lurch updated, thanks for the review :)

**Comment by vipulgupta2048** [comment]
@balena-ci retest

**Comment by lurch** [comment]
Interesting that this update now says that Node12 is the minimum supported version, yet one of the CI checks is named `ResinCI/npm/linux-alpine-x86_64-node10` :wink:

**Comment by nghiant2710** [comment]
@balena-ci I self-certify!

**Comment by 20k-ultra** [comment]
@balena-ci rebase

**Pull request merged**